inner-page.html

```diff
@@ -43,35 +43,36 @@
 
       <div class="row justify-content-center">

```

Comment:
This page itself is not going to be used by us as of now. Remove all the changes made.